builtin/logical/database/backend_test.go

```diff
@@ -66,6 +69,19 @@ func TestBackend_PluginMain_Postgres(t *testing.T) {
 	v5.Serve(dbType.(v5.Database))
 }
 
+func TestBackend_PluginMain_PostgresMultiplexed(t *testing.T) {
+	if os.Getenv(pluginutil.PluginVaultVersionEnv) == "" {
+		return
+	}
+
+	dbType, err := postgresql.New()
+	if err != nil {
+		t.Fatalf("Failed to initialize postgres: %s", err)
+	}
+
+	v5.ServeMultiplex(dbType.(v5.Factory))
```

Comment:
Since `New` _is_ the factory function, I think that you can pass that in directly (same with the other test plugins).
```suggestion
	v5.ServeMultiplex(postgresql.New)
```